Packs/CTIX/Integrations/CTIXv3/CTIXv3.py

```diff
@@ -464,93 +467,101 @@ def tag_indicator_updation(
-        '''
+        """
         url_suffix = "ingestion/tags/"
         client_url = self.base_url + url_suffix
         return self.get_http_request(client_url, **params)
 
     def get_indicator_details(self, object_type: str, object_id: str, params: dict):
-        '''
+        """
         Get Indicator Details
 
         :param str object_type: Object type of the IOCs
         :param str object_id: Object ID of the IOCs
         :param dict params: Paramters to be added in request
         :return dict: Returns response for query
-        '''
+        """
         url_suffix = f"ingestion/threat-data/{object_type}/{object_id}/basic/"
         client_url = self.base_url + url_suffix
         return self.get_http_request(client_url, **params)
 
     def get_indicator_tags(self, object_type: str, object_id: str, params: dict):
-        '''
+        """
         Get Indicator Tags
 
         :param str object_type: Object type of the IOCs
         :param str object_id: Object ID of the IOCs
         :param dict params: Paramters to be added in request
         :return dict: Returns response for query
-        '''
+        """
         url_suffix = f"ingestion/threat-data/{object_type}/{object_id}/quick-actions/"
         client_url = self.base_url + url_suffix
         return self.get_http_request(client_url, **params)
 
     def get_indicator_relations(self, object_type: str, object_id: str, params: dict):
-        '''
+        """
         Get Indicator Relations
 
         :param str object_type: Object type of the IOCs
         :param str object_id: Object ID of the IOCs
         :param dict params: Paramters to be added in request
         :return dict: Returns response for query
-        '''
+        """
         url_suffix = f"ingestion/threat-data/{object_type}/{object_id}/relations/"
         client_url = self.base_url + url_suffix
         return self.get_http_request(client_url, **params)
 
     def get_indicator_observations(self, params: dict):
-        '''
+        """
         Get Indicator Observations
 
         :param dict params: Paramters to be added in request
         :return dict: Returns response for query
-        '''
+        """
         url_suffix = "ingestion/threat-data/source-references/"
         client_url = self.base_url + url_suffix
         return self.get_http_request(client_url, **params)
 
     def get_conversion_feed_source(self, params: dict):
-        '''
+        """
         Get Conversion Feed Source
 
         :param dict params: Paramters to be added in request
         :return dict: Returns response for query
-        '''
+        """
         url_suffix = "conversion/feed-sources/"
         client_url = self.base_url + url_suffix
         return self.get_http_request(client_url, **params)
 
     def get_lookup_threat_data(
-        self, object_type: str, object_names: list, params: dict
+        self, object_type: str, ioc_type: str, object_names: list, params: dict
```

Comment:
Expecting an str means length of it will be the length of the string.
Assuming this may be a mistake and it should be a list, as you check its length after.

Reply:
Fixed.